mmv1/products/alloydb/Instance.yaml

```diff
@@ -236,6 +236,35 @@ properties:
       - !ruby/object:Api::Type::Integer
         name: 'queryPlansPerMinute'
         description: 'Number of query execution plans captured by Insights per minute for all queries combined. The default value is 5. Any integer between 0 and 20 is considered valid.'
+  - !ruby/object:Api::Type::NestedObject
+    name: 'observabilityConfig'
+    description: 'Configuration for enhanced query insights.'
+    default_from_api: true
```

Comment:
Just wanted to double-check that this has a server-side default? If it just returns an empty object (and that will continue to be the case), you could potentially handle that with a [custom decoder or flattener](https://googlecloudplatform.github.io/magic-modules/develop/custom-code/#decoder). default_from_api has some [drawbacks](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#default_from_api) so it's nice to avoid where possible

Reply:
No, it has default values and doesnt send empty config.